runtimevar/runtimeconfigurator/runtimeconfigurator.go

```diff
@@ -106,19 +113,36 @@ type Options struct {
 // state implements driver.State.
 type state struct {
 	val        interface{}
+	raw        *pb.Variable
 	updateTime time.Time
-	raw        []byte
+	rawBytes   []byte
 	err        error
 }
 
+// Value implements driver.State.Value.
 func (s *state) Value() (interface{}, error) {
 	return s.val, s.err
 }
 
+// UpdateTime implements driver.State.UpdateTime.
 func (s *state) UpdateTime() time.Time {
 	return s.updateTime
 }
 
+// As implements driver.State.As.
+func (s *state) As(i interface{}) bool {
+	if s.raw == nil {
+		return false
+	}
+	switch p := i.(type) {
```

Comment:
Why is this done differently from the others?

Reply:
I think because I copy/pasted from `paramstore`, which supports multiple types so used a `switch` instead of a type assertion. Fixed.